Orange/widgets/unsupervised/owdistancetransformation.py

```diff
@@ -52,6 +52,8 @@ def __init__(self):
 
         gui.auto_commit(self.controlArea, self, "autocommit", "Apply",
                         checkbox_label="Apply on any change")
+        gui.separator(box)
+        box.layout().addWidget(self.report_button)
```

Comment:
The button is in the wrong box. You need `box = gui.auto_commit( ... ... ...)`, as in that other commit.